src/engines-experimental/stree.cc

```diff
@@ -33,130 +33,157 @@
 #include <iostream>
 #include <unistd.h>
 
-#include <libpmemobj++/transaction.hpp>
 #include <libpmemobj++/make_persistent_atomic.hpp>
+#include <libpmemobj++/transaction.hpp>
 
 #include "stree.h"
 
 #define DO_LOG 0
-#define LOG(msg) if (DO_LOG) std::cout << "[stree] " << msg << "\n"
+#define LOG(msg)                                                                         \
+	if (DO_LOG)                                                                      \
+	std::cout << "[stree] " << msg << "\n"
```

Comment:
FTR, this is wrong - it will fail if this macro is used like this: if (something) LOG(); else something else;
Don't fix this in this PR. I just randomly stumbled upon this.